op_builder/comm.py

```diff
@@ -0,0 +1,37 @@
+#NOTE: This work is still experimental. Please do not run custom NCCL/MPI deepspeed comm backends until they are fully supported.
+
+import torch
+from .builder import CUDAOpBuilder
+
+MV2 = f'/opt/mvapich2/gdr/2.3.6/no-mpittool/no-openacc/cuda11.3/mofed5.4/mpirun/gnu9.3.0'
+OMPI = f'/usr/local/mpi'
+
+MPI_INCL = OMPI + '/include'
+MPI_LIB = '-L' + OMPI + '/lib -lmpi'
+
+CUDA = '/usr/local/cuda'
```

Comment:
We should pull the CUDA path from torch like this: https://github.com/microsoft/DeepSpeed/blob/6719b46bd8792755cfc75eaa448bf7fa65fd99eb/op_builder/builder.py#L39-L40

You'll need to do this from within the class below, so it's delayed to when the op is built.